index.html

```diff
@@ -285,6 +285,11 @@ <h1 class="white-header">FOSSASIA on social media</h1>
                         </div>
                     </div>
                 </div>
+				<!--Flickr-->
+				<div class="row">
+					<div id="flickrembed"></div>
```

Comment:
Only columns (`.col-*-*`) may be children of `.row`s

*BootLintBear, severity MAJOR, section `html`.*